source/appModules/explorer.py

```diff
@@ -1,9 +1,9 @@
 # -*- coding: UTF-8 -*-
-#appModules/explorer.py
-#A part of NonVisual Desktop Access (NVDA)
-#Copyright (C) 2006-2019 NV Access Limited, Joseph Lee, Łukasz Golonka
-#This file is covered by the GNU General Public License.
-#See the file COPYING for more details.
+# appModules/explorer.py
+# A part of NonVisual Desktop Access (NVDA)
+# Copyright (C) 2006-2019 NV Access Limited, Joseph Lee, Łukasz Golonka, Julien Cochuyt

```

Comment:
This should be 2006-2020. I really think we need to implement automated testing of copyright headers at some point.

Reply:
> This should be 2006-2020.
Done: 93aa7bb

>  I really think we need to implement automated testing of copyright headers at some point.

Probably not before they are all automatically amended and standardized, which is planned but clearly not a priority. Once done, yes, I agree.